lib/libexfat.c

```diff
@@ -160,7 +163,24 @@ int exfat_get_blk_dev_info(struct exfat_user_input *ui,
 		goto out;
 	}
 
+	if (fstat(fd, &st) == 0 && S_ISBLK(st.st_mode)) {
+		char pathname[43];
+		FILE *fp;
+		snprintf(pathname, sizeof pathname, "/sys/dev/block/%u:%u/start",
+			major(st.st_rdev), minor(st.st_rdev));
+		if ((fp = fopen(pathname, "r")) != NULL) {
```

Comment:
do not use assignment in if condition.

Reply:
exfatprogs have been written with kernel coding style. please check above 3 comment.